Games/types/Mafia/roles/cards/MindShifter.js

```diff
@@ -0,0 +1,50 @@
+const Card = require("../../Card");
+const { PRIORITY_EFFECT_GIVER_DEFAULT } = require("../../const/Priority");
+
+module.exports = class MindShifter extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Warp Mind": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive"], exclude: ["Monsters"] },
+                action: {
+                    labels: ["effect"],
+                    priority: PRIORITY_EFFECT_GIVER_DEFAULT,
+                    run: function () {
+                        this.actor.role.data.insane = this.target;
+                        this.target.queueAlert("You will be driven insane if not visited by a player not aligned with the Monsters, tonight!");

```

Comment:
if it's tonight then the priority is wrong, should be +1

Reply:
the player receives this message the follwing day

timeline would look like:

N0 - picks person
D1 - "you will be driven insane if not visited tonight"
N1 - action runs during night
D2 - determines if player becomes insane or not

Reply:
I see, the action is meant for the next night. Thanks!